gitstrap.go

```diff
@@ -340,6 +320,19 @@ func (strap *strapCtx) addCollaborators(owner string, repo *github.Repository) e
 	return nil
 }
 
+func getOwner(strap *strapCtx, opt Options) (string, error) {
```

Comment:
@vassilevsky let's put a doc string with small overview what that function can return  (organization name if specified or current user name)

Reply:
Added.